modules/trusted-firmware-m/Kconfig.tfm

```
@@ -310,6 +310,31 @@ config TFM_MCUBOOT_PATH_DOWNLOAD
 
 endchoice
 
+choice TFM_ETHOS_DRIVER_PATH
+	prompt "Use Zephyr's local ETHOS driver path or DOWNLOAD to fetch automatically"
```

Rather something like:
```suggestion
	prompt "Source used for the Ethos-U driver"
```
(Not saying my phrasing is perfect, but yours just says "use this or that", which is not how choice Kconfig options are phrased.)